### PR TITLE
Move financial acl setting to the extension

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Move financial acl setting to the extension

Before
----------------------------------------
Setting in main settings

After
----------------------------------------
Setting in the extension

Technical Details
----------------------------------------
The onChange is mostly to support unit tests 

image

Comments
----------------------------------------

### CODE REVIEW

File: tests/phpunit/CRMTraits/Financial/FinancialACLTrait.php [hunk 1]
```diff
@@ -20,24 +20,18 @@ trait CRMTraits_Financial_FinancialACLTrait {
    * Enable financial ACLs.
    */
   protected function enableFinancialACLs() {
-    $contributeSettings = Civi::settings()->get('contribution_invoice_settings');
     $this->callAPISuccess('Setting', 'create', [
-      'contribution_invoice_settings' => array_merge($contributeSettings, ['acl_financial_type' => TRUE]),
```
Comment (eileenmcnaughton) [inline tests/phpunit/CRMTraits/Financial/FinancialACLTrait.php:25]: this was actually the first setting we separated out of the contribution_invoice_settings borg & has been a free agent even longer than the rest

**Comment by eileenmcnaughton** [comment]
@seamuslee001 I think this should be an easy merge = it's a move + some test fixes around caching

**Comment by seamuslee001** [comment]
Jenkins re test this please

**Comment by seamuslee001** [comment]
I agree with this, just getting Jenkins to re-run given its been a bit of time since it was last run

**Pull request merged**